### DISCUSSION TITLE
Create files in srcDir

### DISCUSSION THREAD

**Discussion by go4cas** [nuxtrdev/nuxtr-vscode#9] [General]
Thanks for the great extension!

I normally set the `srcDir` in `nuxt.config.ts` to `src/`. But, when using Nuxtr to create new files, it will add these to the project root. Is there a way to get Nuxtr to rather use the `srcDir`?

**Comment by adhamfarrag** [comment]
Hi @go4cas ! 

Thanks for your kind words. 
This is already fixed and should be available in the next release. :)

**Comment by adhamfarrag** [comment]
Hello @go4cas 

`srcDir` and monorepo are now supported. Read more on them here https://github.com/nuxtrdev/nuxtr-vscode#srcdir-and-monorepo-support

Thanks for suggesting this feature :)

**Comment by go4cas** [comment]
This works really well! Thanks for the update! Keep up the amazing work!
